### ISSUE TITLE
Light alerts fail contrast minimum

### ISSUE THREAD

**Issue by patrickhlauke** [twbs/bootstrap#32052]
`.alert-light` alerts currently use foreground `#818182` and background `#FEFEFE` with contrast ratio 3.9:1. text should be darkened a smidge to reach 4.5:1

https://v5.getbootstrap.com/docs/5.0/components/alerts/

> ![contrast-bootstrap-alerts](https://user-images.githubusercontent.com/895831/97921918-8c674000-1d53-11eb-8807-599e46c2e85f.png)

**Comment by StarTrooper08** [comment]
Can I work on this issue.
I can solve this contrast issue.

**Comment by Deckluhm** [comment]
> May I work on this issue.
> I can correct the contrast issue.

As an answer to ^ and https://github.com/twbs/bootstrap/discussions/32061, you sure can propose a fix to this issue by opening a pull request, as long as you are following [contributing guidelines](https://github.com/twbs/bootstrap#contributing).
